assets/capability/capabilities/alarm_pm25.json

```diff
@@ -2,7 +2,7 @@
   "type": "boolean",
   "title": {
     "en": "PM2.5 alarm",
-    "nl": "PM2.5 alarm",
+    "nl": "PM2,5-alarm",
```

Comment:
```suggestion
    "nl": "PM2.5-alarm",
```

Reply:
Dit moet gewoon 2.5 zijn hoor, erg verwarrend als we in het Nederlands opeens een komma gaan gebruiken

Reply:
Dit is wel hoe het in het Nederlands hoort, aangezien het decimalen zijn